test/QuLDE_tests.jl

```diff
@@ -33,4 +33,15 @@ end
     out = solve(qprob, QuLDE(), k)
 
     @test isapprox.(s[end-1:end], out, atol = 0.01) |> all
+
+    qprob = QuLDEProblem(An, b, x, tspan)
+    f(u,p,t) = An*u + b;
+    prob = ODEProblem(f, x, tspan)
+
+    sol = solve(prob, Tsit5(), dt = 0.1, adaptive = :false)
+    s = vcat(sol.u...)
+
+    out = solve(qprob, QuLDE(), k)
+
+    @test isapprox.(s[end-1:end], out, atol = 0.1) |> all
```

Comment:
sorry I need to catch up a little bit, which paper are you implementing in this PR?

Reply:
https://arxiv.org/abs/1807.04553.
Appendix A , A is non-Unitary :